shared/src/types/api/statistics.dto.ts

```diff
@@ -1,19 +1,19 @@
 import { Difficulty } from "../base";
 
 export type UserStatisticsRes = {
-  attemptSummary: Record<Difficulty, number>;
+  attemptSummary: Record<Difficulty | string, number>;
```

Comment:
String enums are a limited range of strings by default.
```suggestion
  attemptSummary: Record<Difficulty, number>;
```